apps/docs/content/components/River/index.mdx

```diff
@@ -174,6 +177,12 @@ Always use high quality images and videos. Avoid using images with text or singl
 
 The image is automatically styled to fit the width of the parent container. If you have bespoke styling requirements, you can disable the provided default styles and style the image manually.
 
+### Additional guidance
+
+- We recommend using a river left as the first river of the flow to start with a visual and avoid stacking text with above section on mobile
+- River components stacks create a unit, the set should be considered to be the main parent section. The parent section should be spaced as a regular section with the largest spacing while internal elements should use smaller spacing
+- If the image is critical for the user to understand the message, default to right aligned. If the text is critical for the user to understand the image, choose left aligned.
```

Comment:
Fine like this for now, but I do think these would make for some **great** visual do/don't examples that can sit under the stacking section above.

Reply:
Ya, that last section `Additional guidance` feels a bit too abstract and could help some definition via a few visuals and be meshed with the other section. Will do another PR to address this.